packages/form-js-viewer/assets/form-js.css

```diff
@@ -212,8 +308,16 @@
 
 .fjs-container .fjs-form-field.fjs-has-errors .fjs-input,
 .fjs-container .fjs-form-field.fjs-has-errors .fjs-select,
-.fjs-container .fjs-form-field.fjs-has-errors .fjs-textarea {
+.fjs-container .fjs-form-field.fjs-has-errors .fjs-textarea,
+.fjs-container .fjs-form-field.fjs-has-errors .fjs-input-group,
+.fjs-container .fjs-form-field.fjs-has-errors .fjs-input-group .fjs-input {
   border-color: var(--color-warning);
+  outline-color: var(--color-warning);
```

Comment:
Shouldn't we rely on the browser/integrator default?

Reply:
The outlines are entirely bespoke in form-js. We basically re-implement them everywhere. So I'd say not. 

Also the border and outline colors need to match, this is something that was missing before and led to not so pretty highlights.